### PR TITLE
Signup and Login page

### PR DESCRIPTION
**Pull request by anamkaa**
Added Signup And Login Pages
Screenshot (298)
Screenshot (299)

### CODE REVIEW

File: stylesheets/page/login.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
+@import url("/stylesheets/style.css");
+
+@import url("/stylesheets/style.css");
```
Comment (dev-enforced) [inline stylesheets/page/login.css:3]: Remove the extra line

File: stylesheets/page/signup.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+.gh-input-wrap-col {
+  display: flex;
+  flex-direction: column;
+}
+
+.gh-checkbox-text {
+  font-size: 0.8rem;
+  margin: 0 0.3rem 0 0.3rem;
```
Comment (dev-enforced) [inline stylesheets/page/signup.css]: You can use margin shorthand property too for handling this. Apply the same in login.css file too

File: stylesheets/page/login.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  width: 22rem;
+  background-color: #f8f9fa;
+  border-radius: 0.2rem;
+  box-shadow: rgb(0 0 0 / 2%) 0px 1px 1px, rgb(0 0 0 / 2%) 0px 4px 4px,
+    rgb(0 0 0 / 2%) 0px 4px 4px, rgb(0 0 0 / 2%) 0px 8px 8px,
+    rgb(0 0 0 / 2%) 0px 16px 16px;
+}
+
+.gh-input-margin {
+  margin: 1rem 3rem 1rem 3rem;
```
Comment (dev-enforced) [inline stylesheets/page/login.css]: Use margin shorthand property for this !!
margin:1rem 3rem; will also work

File: stylesheets/page/signup.css (hunk 2 above)
Comment (dev-enforced) [reply stylesheets/page/signup.css]: Can use px too if 0.3 rem gets less than 10px

**Review by dev-enforced — APPROVED** [review]
The code looks good to me. Good work Anamika!!
You can store all the common styles in one file (authentication.css) because in login and signup some of the classes were getting repeated
Do deploy this on Netlify and attach the link in PR so that the previewers can get an idea on real computer screen as well

File: stylesheets/page/login.css (hunk 1 above)
Comment (anamkaa) [reply stylesheets/page/login.css:3]: Will do that.

File: stylesheets/page/signup.css (hunk 2 above)
Comment (anamkaa) [reply stylesheets/page/signup.css]: Done , as suggested.

File: stylesheets/page/login.css (hunk 3 above)
Comment (anamkaa) [reply stylesheets/page/login.css]: Done , as suggested.

File: page_layouts/login.html [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+          <label for="gh-password" class="gh-input-text">Password</label>
+          <input type="password" id="gh-password" class="gh-input" />
+        </div>
+
+        <div class="gh-input-wrap-row gh-input-margin">
+          <div class="gh-input-wrap-row">
+            <input type="checkbox" name="" id="accept" class="gh-checkbox" />
+            <label for="accept" class="gh-checkbox-text">Remember me</label>
+          </div>
+          <div class="gh-green-text">Forgot your Password?</div>
```
Comment (shubhamkumar648) [inline page_layouts/login.html]: you can also use span ,p ,h tag for small text

File: stylesheets/page/login.css [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.gh-green-text {
+  margin-left: auto;
+  font-size: 0.7rem;
+  color: #98a86e;
```
Comment (shubhamkumar648) [inline stylesheets/page/login.css]: you can give value of color in variable

File: stylesheets/page/signup.css [hunk 6]
```diff
@@ -0,0 +1,80 @@
+@import url("/stylesheets/style.css");
+
+body {
+  background-color: #f2f3f4;
```
Comment (shubhamkumar648) [inline stylesheets/page/signup.css]: same here use var for color value

**Review by shubhamkumar648 — APPROVED** [review]
just consider my comment as a suggestion! remaining good use of utils class name, good design

File: stylesheets/page/signup.css (hunk 6 above)
Comment (anamkaa) [reply stylesheets/page/signup.css]: done , as suggested.

File: stylesheets/page/login.css (hunk 5 above)
Comment (anamkaa) [reply stylesheets/page/login.css]: done.

**Pull request merged**